function.js

```diff
@@ -0,0 +1,20 @@
+// Implementing  Functions 
+console.log("hello, world");
+
+//Implemented Anonymous Function
+const add= function(a,b){
+    return a+b;
+}
+
+const multi = function(a,b){
+    return a*b;
+}
+
+// Implemented Normal Function
+function sub(){
+    return a-b;
+}
+//Implemented Arrow Function 
+const div = (a,b) => a/b;
+
+const mod = (a,b)=> a%b;
```

Comment:
make sure that the names of variables and functions should be meaningful. here we should name these functions as addition, multiplication, subtraction, modulo, and division instead of add, multi, sub, div, mod

Reply:
Resolved issue `6b74bc8a80` , `8d2e8856`